server/models/User.py

```diff
@@ -15,10 +13,10 @@ def load_user(user_id):
 
 class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
+    public_id = db.Column(db.String(50), unique=True)
```

Comment:
What would be the difference from this to the `id`?

Reply:
`id` field will contain consecutive numbers starting from 1. It can be used while doing array operations but `public_id` is used to store a unique but considerably large alpha-numeric string. So, I think this is great to be a primary key for the User table.

Tbh there won't be much need for the `id` field. We can discard it.
Your suggestions? @JPTIZ

Reply:
Well, for a 50-lengthed string I think it won't have too much chances of collision for this project's user base length, so I think the change will be fine (as even if a collision happens, we could just regenerate the number once more, and we shouldn't be doing that too much).

We can then just let `id` become what `public_id` is now (iow discard the current `id` field and rename `public_id` to `id).

Reply:
> Well, for a 50-lengthed string I think it won't have too much chances of collision for this project's user base length, so I think the change will be fine (as even if a collision happens, we could just regenerate the number once more, and we shouldn't be doing that too much).
> 
> We can then just let `id` become what `public_id` is now (iow discard the current `id` field and rename `public_id` to `id).

Roger that!
I'll commit & push the changes now itself